src/Microsoft.Extensions.Logging.Console/ConsoleLoggerOptions.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.Extensions.Logging.Console
     public class ConsoleLoggerOptions
     {
         public bool IncludeScopes { get; set; } = false;
+        public bool DisableColors { get; set; } = false;
```

Comment:
No need to set default "default values". I'd recommend removing the ` = false;` from this line *and* the one above it.

Reply:
Addressed in 2bfbdfb2d5356bc7145a47dff05f953b99b76279